packages/mdc-list/README.md

```diff
@@ -134,7 +134,7 @@ OR
 ### Single Selection List
 
 MDC List can handle selecting/deselecting list elements based on click or keyboard action. When enabled, the `space` and `enter` keys (or `click` event) will trigger an 
```

Comment:
> selecting/deselecting

is this still relevant?

Reply:
Yes, when clicked it will deselect the previously selected list item and select the currently selected list item.